### PR TITLE
add rio-cogeo and supermercado

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge/staged-recipes this ready for review.

### CODE REVIEW

**Comment by scopatz** [comment]
Seems to fail on linux :frowning_face:

**Comment by ocefpaf** [comment]
> Seems to fail on linux

Interesting. It passed locally without `contextlib2`. I love `pkg_resources`!

**Comment by ocefpaf** [comment]
@scopatz I'm sacrificing Python 2.7 here for the sake of `noarch` builds. Can you take a second look and merge it?

File: recipes/rio-cogeo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >3.3
+    - pip
+    - rasterio[s3] >=1.0.9
```
Comment (scopatz) [inline recipes/rio-cogeo/meta.yaml]: Does using square brakets here really work?

Comment (ocefpaf) [reply recipes/rio-cogeo/meta.yaml]: Nope. That is a remnant from conda-skeleton. I added the proper dependency below. I'm surprised it did not crash the build though.

Comment (scopatz) [reply recipes/rio-cogeo/meta.yaml]: me too!

**Comment by scopatz** [comment]
Thanks @ocefpaf!

**Pull request merged**